### ISSUE TITLE
"sf package ..." command fails with: JitPluginInstallError: Could not install @salesforce/plugin-packaging Code: JitPluginInstallError

### ISSUE THREAD

**Issue by jovan3os** [forcedotcom/cli#2047]
I'm getting this error when running a "sf package version create ..." command. 

I did the following things in attempt to fix this: 

1. sfdx update (ran successfully)
2. sf update (ran successfully)
3. sfdx plugins-install @salesforce/plugin-packaging (ran successfully)
4. "sf plugins install @salesforce/plugins-packaging", and that fails with

CannotFindNodeExecutable: Cannot locate node executable.
Code: CannotFindNodeExecutable

So I assume the latter is the root of the problem. I could not find anything online about this error, nor I'm very familiar with "node".

Thank you for your help :)

**Comment by WillieRuemmele** [comment]
Hey @jovan3os - JIT, or just-in-time, plugin installation is how we are shipping the "packaging" commands. What's the output of `sf plugins --core`? What happens if you just try running a `sf packaging *` command? that should initiate an installation of the packaging plugin

I also just want to confirm that you ran `sf plugin install plugin-packaging` - your instruction had an extra `s`... but it was able to install for me...
```
 ➜  sf plugins install @salesforce/plugin-packaging 
Polling for new version(s) to become available on npm... done
Successfully validated digital signature for @salesforce/plugin-packaging.
Finished digital signature check.
Installing plugin packaging... ⡿ [1/4] Resolving packages...
Installing plugin packaging... ⣾ [2/4] Fetching packages...
warning "@salesforce/plugin-packaging > @oclif/core > ts-node@10.9.1" has unmet peer dependency "@types/node@*".
Installing plugin packaging... installed v1.16.5
```

and with the JIT installation...
```
 ➜  sf package version list                         
Polling for new version(s) to become available on npm... done
Successfully validated digital signature for @salesforce/plugin-packaging.
Finished digital signature check.
Installing plugin packaging... ⣾ [1/4] Resolving packages...
Installing plugin packaging... ⢿ [2/4] Fetching packages...
warning "@salesforce/plugin-packaging > @oclif/core > ts-node@10.9.1" has unmet peer dependency "@types/node@*".
Installing plugin packaging... installed v1.16.3
=== Package Versions [153]
```

**Comment by jovan3os** [comment]
Hi @WillieRuemmele , 

I figured out the issue, and it was because I was missing the node installation on my Mac. Although the error **kind of** suggests that, I think it'd be good to improve that error message to say something along the lines of "node installation missing or $PATH not set". Especially when there's nothing on Google that suggests fixing this error. 

Here's the output of my `sf plugins --core` before I installed the package

`@oclif/plugin-autocomplete 2.1.6 (core)
@oclif/plugin-commands 2.2.11 (core)
@oclif/plugin-help 5.2.8 (core)
@oclif/plugin-not-found 2.3.22 (core)
@oclif/plugin-plugins 2.4.3 (core)
@oclif/plugin-search 0.0.14 (core)
@oclif/plugin-update 3.1.8 (core)
@oclif/plugin-version 1.3.1 (core)
@oclif/plugin-warn-if-update-available 2.0.32 (core)
@oclif/plugin-which 2.2.17 (core)
@salesforce/cli 1.71.0 (core)
apex 2.2.7 (core)
auth 2.7.10 (core)
data 2.3.8 (core)
deploy-retrieve 1.8.5 (core)
env 2.1.3 (core)
info 2.6.2 (core)
limits 2.3.9 (core)
login 1.2.4 (core)
org 2.6.2 (core)
schema 2.3.4 (core)
settings 1.4.2 (core)
sobject 0.1.9 (core)
telemetry 2.1.3 (core)
templates 55.4.4 (core)
trust 2.4.5 (core)
user `
